modules/accredited_representative_portal/db/migrate/20250313190720_create_ar_icn_temporary_identifiers.rb

```diff
@@ -0,0 +1,9 @@
+class CreateArIcnTemporaryIdentifiers < ActiveRecord::Migration[7.2]
+  def change
+    create_table :ar_icn_temporary_identifiers, id: :uuid do |t|
+      t.string :icn_ciphertext, null: false
```

Comment:
We can't encrypt this bit because we want to do joins and search against this value.